console/my-account/how-to/create-an-account.mdx

```diff
@@ -13,43 +13,29 @@ dates:
-
-5. Click the verification link in your mailbox to confirm your email address and agree to our terms of service. You are redirected to the Scaleway Console. A welcome message displays.
-  <Lightbox src="scaleway-welcome-aboard-modal.webp" alt="" />
-  You are asked to select between three account levels:
-  
-    - **Discover:** allows you to browse through the Scaleway console and discover the catalog without ordering.
-    - **Test:** allows you to order a few resources and test them before starting development or production.
-    - **Build:** allows you to order any type of resources and scale your project as it grows.    
-  
   <Message type="note">
 
-  To unlock the different levels, you will need to progressively add information to your account.
+  If you are a startup, you can create a corporate account and check **Yes** under **Are your a startup?**. This allows you to benefit from offers dedicated to startups.
 
   </Message>
-  
-    |                      |         Discover         |           Test           |           Build          |
-    |----------------------|:------------------------:|:------------------------:|:------------------------:|
-    | Email address        | <Icon name="validate" /> | <Icon name="validate" /> | <Icon name="validate" /> |
-    | Billing information  |                          | <Icon name="validate" /> | <Icon name="validate" /> |
-    | Identity check (KYC) |                          |                          | <Icon name="validate" /> |
 
-6. Select an account level and click **Get started!**.
+4. Check the box and solve the [reCaptcha](https://en.wikipedia.org/wiki/ReCAPTCHA) if you would like to receive marketing communications from Scaleway. Then, click **Get Started!** to create your account. A message displays to request that you check your email inbox for the verification email.
 
-The next steps will vary, depending on the level you chose. Follow the links relevant to your case for further instructions:
+5. Click the verification link in your mailbox to confirm your email address and agree to our terms of service. You are redirected to the Scaleway Console. A welcome message displays.
+
+  <Lightbox src="scaleway-welcome-aboard-modal.webp" alt="" />
 
-- If you chose **Discover**, you can browse the Scaleway console without needing to take any further action yet.
-- If you chose **Test**, you will be prompted to [add billing information](/console/my-account/how-to/manage-billing-information/) to your account
-- If you chose **Build**, you will be prompted to [add billing information](/console/my-account/how-to/manage-billing-information/) to your account, and to [verify your identity](/console/my-account/how-to/verify-identity/)
+6. Click **Add information** to [add your billing information](/console/my-account/how-to/manage-billing-information/#how-to-add-a-payment-method) and payment method. This will give you access to the Scaleway resources. 
```

Comment:
```suggestion
6. Click **Add information** to [add your billing information](/console/my-account/how-to/manage-billing-information/#how-to-add-a-payment-method) and payment method. This will give you access to Scaleway resources. 
```